tfjs-converter/python/tensorflowjs/converters/fuse_prelu.py

```diff
@@ -197,4 +200,6 @@ def prelu_fn(*args):
   with graph.as_default():
     prelu_fn(tf.constant(1.0), tf.constant(1.0))
 
-register_prelu_op()
+
+if tf.version.GIT_VERSION != 'unknown' and tf.__version__ <= '2.1':
```

Comment:
This includes both 2.0 and 2.1. Is that intentional?

Reply:
Nope, changed to <. Thanks for spotting that!